solr/solrj/src/test/org/apache/solr/client/solrj/impl/CloudHttp2SolrClientTest.java

```diff
@@ -268,35 +258,90 @@ public void testAliasHandling() throws Exception {
   }
 
   @Test
-  @LogLevel("org.apache.solr.servlet.HttpSolrCall=INFO")
+  @LogLevel("org.apache.solr.servlet.HttpSolrCall=DEBUG")
   public void testHttpCspPerf() throws Exception {
 
-    String COLLECTION = "TEST_COLLECTION";
-    CollectionAdminRequest.createCollection(COLLECTION, "conf", 2, 1)
-        .process(cluster.getSolrClient());
-    cluster.waitForActiveCollection(COLLECTION, 2, 2);
+    performTest(false, "HTTPCSPTEST",
+            1,
+            10,
+            //1 create collection, 2 /admin/core , 1 LISTALIASES, 1 CLUSTERSTATUS, 10 CLUSTERSTATUS with coll param
```

Comment:
I propose that collection creation be external to the logging observation.  Create it beforehand.  The reason why is that it'll generate a number of calls including internal distributed ones that are not what we're trying to observe (we're only trying to observe requests *from* our client, not from the Overseer or whatever Solr node.

Reply:
Done